src/js/actions/layers.js

```diff
@@ -151,15 +159,20 @@ define(function (require, exports) {
             optionalPropertiesPromise = descriptor.batchMultiGetProperties(references, optionalProperties,
                 { continueOnError: true });
 
-        var nameSpace = global.EXTENSION_DATA_NAMESPACE,
-            extensionPlayObjects = references.map(function (ref) {
-                return layerLib.getExtensionData(ref[0], ref[1], nameSpace);
-            }),
+        var extensionPromise;
+        if (lazy) {
+            extensionPromise = Promise.resolve();
```

Comment:
To test this case, note that all text layers (including non-selected ones) are reset when the font list is first initialized, and that happens when a text layer is first selected.